scripts/commands/godmode.lua

```diff
@@ -1,16 +1,17 @@
 -----------------------------------
 -- func: godmode
 -- desc: Toggles god mode on the player, granting them several special abilities.
+-- Pass variable of 1 to command to enable a "soft" god mode.
 -----------------------------------
 
 cmdprops =
 {
     permission = 1,
-    parameters = ""
+    parameters = "i"
 }
 
-function onTrigger(player)
-    if (player:getCharVar("GodMode") == 0) then
+function onTrigger(player, arg1)
```

Comment:
'arg1' doesn't really tell us what this is doing. Try something like 'mode'

Reply:
"tier" works

Reply:
Adjusted to "tier"